lib/load-migrations.js

```diff
@@ -7,6 +7,43 @@ const Migration = require('./migration')
 
 module.exports = loadMigrationsIntoSet
 
+async function loadFromMigrationsDirectory (migrationsDirectory, filterFn) {
```

Comment:
I notice a few unrelated changes made in here. Was there a reason to make changes and not just copy them out? Ideally we would just copy them out as is. I would not block on this, but for future reference, keeping PRs to the minimal related changes is best so we don't have to spend time digging around for what changed.

Reply:
Previously this part would modify the `migMap` object/map ;) as well as maintain an array of `Migration` objects. So I had to refactor this part since it's extracted into a function now and should have one return output, which is `migMap` object.

Reply:
oh hm. I was talking about the inline'ing of line 30. Maybe it is good, just meant I had to read each line in this method instead of just skimming it as "copied". Anyway, will mark this as resolved.